net/pfSense-pkg-ntopng/files/usr/local/pkg/ntopng.inc

```diff
@@ -121,6 +121,17 @@ function ntopng_sync_package() {
 				$local_networks = "--local-networks " . escapeshellarg(gen_subnet(get_interface_ip('lan'), get_interface_subnet('lan')) . '/' . get_interface_subnet('lan'));
 			}
 			break;
+		case "custom":
+			$nets = array();
+			foreach ($ntopng_config['row'] as $net) {
+				if (is_subnet($net['cidr'])) {
+					$nets[] = $net['cidr'];
```

Comment:
You're right. I should have thought about this right away.

Fixing it shortly.